chainer/links/caffe/caffe_function.py

```diff
@@ -547,18 +554,6 @@ def __call__(self, *xs):
         return self.func(xs, **self.kwargs)
 
 
-class _DropoutFunction(object):
```

Comment:
Oh, I didn't take care on that. Other changes for v2 would also have pickle incompatibilities as well. I don't think it's reasonable to keep the pickle compatibility for v2, though. How do you think of it?